diff --git a/subversion/bindings/swig/perl/native/Makefile.PL.in b/subversion/bindings/swig/perl/native/Makefile.PL.in
index 3b36901cc..6e45a4cd8 100644
--- a/subversion/bindings/swig/perl/native/Makefile.PL.in
+++ b/subversion/bindings/swig/perl/native/Makefile.PL.in
@@ -38,6 +38,7 @@ my $swig_builddir = "${svnlib_builddir}/bindings/swig";
 
 my @modules = qw/client delta fs ra repos wc/;
 my @ldpaths = (abs_path($swig_builddir) . "/perl/libsvn_swig_perl/.libs",
+               '@prefix@/lib',
                map { abs_path($svnlib_builddir) . "/libsvn_$_/.libs" }
                    @modules, qw/diff subr ra_local ra_svn ra_serf
                                 fs_base fs_util fs_fs fs_x/);
